platform/linux-generic/odp_packet.c

```diff
@@ -166,44 +166,6 @@ static inline void push_tail(odp_packet_hdr_t *pkt_hdr, uint32_t len)
 	last_seg->seg_len  += len;
 }
 
-/* Copy all metadata for segmentation modification. Segment data and lengths
- * are not copied. */
-static inline void packet_seg_copy_md(odp_packet_hdr_t *dst,
-				      odp_packet_hdr_t *src)
-{
-	dst->p = src->p;
-
-	/* lengths are not copied:
-	 *   .frame_len
-	 *   .headroom
-	 *   .tailroom
-	 */
```

Comment:
The new _odp_packet_copy_md() function should also document which fields will not be copied: those that relate to data buffering and are set already by a previous alloc/copy call.